browser_history/cli.py

```diff
@@ -50,10 +59,15 @@ def main():
     It parses arguments from sys.argv and performs the appropriate actions.
     """
     args = parser.parse_args()
+    assert args.type in ['history','bookmarks','all'], \
+                        f"Type should be one of all, {AVAILABLE_TYPES}"
 
-    if args.browser == 'all':
-        outputs = get_history()
+    h_outputs = b_outputs =None
+    fetch_map = {'history':[h_outputs ,get_history()],
+                 'bookmarks':[b_outputs,get_bookmarks()]}
```

Comment:
These functions would get evaluated every time, right? Since you're calling them here.

Reply:
Oh should i put only the function name then?

Reply:
Yes